testpilot/frontend/static-src/app/views/landing-page.js

```diff
@@ -5,6 +5,8 @@ import ExperimentListView from './experiment-list-view';
 
 // TODO back out this commit (78bd0a93a1q) before launch.
 import queryString from 'query-string';
+const installedMsgTmp = `<h2 class="emphasis">Test Pilot installed!</h2>
+<a href="/" class="button large default quick-pop">Let's Go</a>`;
```

Comment:
Yeah, we don't have any translations yet - but, any new text elements really should be marked with a `data-l10n-id` and added to [locales/en-US/app.l20n](https://github.com/mozilla/testpilot/blob/master/locales/en-US/app.l20n)